### PR TITLE
nuke: dont poweroff nodes, leave it as is for adhoc debug if needed.

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
jenkins test this please

**Comment by vasukulkarni** [comment]
jenkins test this please

File: teuthology/nuke/__init__.py [hunk 1]
```diff
@@ -304,8 +304,7 @@ def nuke_helper(ctx, should_unlock):
         check_lock.check_lock(ctx, None, check_up=False)
     status = get_status(host)
     if status['machine_type'] in provision.fog.get_types():
-        remote = Remote(host)
-        remote.console.power_off()
+        log.info("Fog provisioned nodes, leaving it as is")
```
Comment (kshtsk) [inline teuthology/nuke/__init__.py]: What about `--poweroff true|false` option for `teuthology-nuke`.

**Comment by kshtsk** [comment]
@vasukulkarni what is a use case for this feature? Just for onetime debugging or something we want to be merged?

**Comment by kshtsk** [comment]
@vasukulkarni are you going to work on this?

**Comment by kshtsk** [comment]
@vasukulkarni ping

**Comment by vasukulkarni** [comment]
> @vasukulkarni ping

sorry for the delay but I like the idea of --poweroff option that is provided to --suite that modifies this behaviour (the original concern was on the power costs if we leave it on), It does help for debugging when test fails and that was original intention.

**Comment by kshtsk** [comment]
@vasukulkarni so... will you guys work on it?

**Comment by vasukulkarni** [comment]
@rakeshgm what do you think, should we take this up further here?

**Comment by rakeshgm** [comment]
I think its not urgent now. since the machines we are using for running tests are not being used for running tests locally. but the cli option does help.

**Comment by kshtsk** [comment]
Ok, don't you mind to close it for now and open another PR when you need it?

**Comment by kshtsk** [comment]
It would be also great to describe the problem which this will help to resolve in the tracker.

**Comment by kshtsk** [comment]
Since no interest from the author, closing now.

**Pull request closed without merging**